crypto/transactions/builder/base.py

```diff
@@ -66,6 +67,9 @@ def multi_sign(self, passphrase, index):
 
     def schnorr_verify(self):
         return self.transaction.verify_schnorr()
+    
+    def schnorr_verify_second(self):
```

Comment:
this needs to get a `secondPublicKey` parameter that you pass along to the verify function